SPECS/erlang/erlang.spec

```
@@ -1,16 +1,21 @@
 %define  debug_package %{nil}
-Name:         erlang
-Summary:      erlang
-Version:      24.2
-Release:      1%{?dist}
-Group:        Development/Languages
+%define  usr_local_bin %{_prefix}/local/bin
+%define  usr_local_lib %{_prefix}/local/lib
+Summary:        erlang
+Name:           erlang
+Version:        25.2
+Release:        1%{?dist}
+License:        ASL 2.0
```

Can we use an SPDX license expression here?

```suggestion
License:        Apache-2.0
```

---

Updated